### PR TITLE
Install sqlite3 from npm when installing the deb rather than when building it

### LINKED ISSUE #26: In thelounge v3.0-pre5 it doesn't correctly install sqlite
```
Error: Cannot find module '/usr/lib/node_modules/thelounge/node_modules/sqlite3/lib/binding/node-v59-linux-x64/node_sqlite3.node'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/lib/node_modules/thelounge/node_modules/sqlite3/lib/sqlite3.js:4:15)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
```

Note `/usr/lib/node_modules/thelounge/node_modules/sqlite3/lib/binding/node-v57-linux-x64/node_sqlite3.node` does exist, but not v59

### PR DESCRIPTION
**Pull request by astorije**
Fixes #26 (hopefully).

Follow-up of https://github.com/thelounge/thelounge-deb/issues/26#issuecomment-394931842 and https://github.com/thelounge/thelounge-deb/pull/27.
Made possible after https://github.com/thelounge/thelounge/pull/2391 and https://github.com/thelounge/thelounge/pull/2541.
If successful, supersedes #31 (also cherry-picks @xPaw's cleanup commits).

Let's see how that goes...

### CODE REVIEW

**Comment by astorije** [comment]
I don't know wth I did wrong with my last WIP commit, but my laptop refuses to even open the Travis log without crashing.

File: debian/postinst [hunk 1]
```diff
@@ -3,6 +3,8 @@ set -e
 
 [[ "$1" == "configure" ]] || exit 0
 
+sudo npm install -g sqlite3 --unsafe-perm
```
Comment (xPaw) [inline debian/postinst]: We'd want to install it into lounge's folder, not globally. I don't want to rely on global npm install nor leave files all over the place (removing lounge won't remove sqlite3 npm).

Comment (astorije) [reply debian/postinst]: Yeah totally agree!
Do you know how I can do this? I would say with `sudo npm install --unsafe-perm --prefix /usr/lib/node_modules/thelounge sqlite3`, does that seem correct to you?

Comment (xPaw) [reply debian/postinst]: Probably.

Comment (astorije) [reply debian/postinst]: Alright, should be fine now. And it even passes CI 🤷‍♂️

**Comment by astorije** [comment]
> I asume all the FIXME code will go away with next lounge prerelease?

Yep exactly. In a separate commit so I'll just remove that commit entirely.

**Comment by astorije** [comment]
@xPaw, I prepped this PR for pre.7, could you confirm it's still okay to you? It's late and I don't want to merge 💩 :)

**Comment by xPaw** [comment]
Okay let's try this.

**Pull request merged**